game_eggs/minecraft/java/quilt/egg-quilt.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-07-08T16:11:20+03:00",
+    "name": "Quilt",
+    "author": "diedyesterdaywashere@gmail.com",
+    "description": "The Quilt project is an open-source, community-driven modding toolchain designed primarily for Minecraft. By focusing on speed, ease of use and modularity, Quilt aims to provide a sleek and modern modding toolchain with an open ecosystem.",
+    "features": null,
+    "docker_images": {
+        "Java 8": "ghcr.io\/pterodactyl\/yolks:java_8",
```

Comment:
i would only use the evrsion, what is really needed.

Reply:
alr give me a bit to re-export the egg

Reply:
actually, i think minecraft can be ran with all of these versions, but i can remove j11 because i haven't really seen any versions that only support it @gOOvER

Reply:
the problem is; there are users who choose something and then it doesn't work :D and then they ask for support even though they used the wrong java version. 

In my opinion you should only use the latest version; all others are EOL.

But this is just my opinion. I don't play Minecraft :D

Reply:
you're kinda right, but the problem is that everything till 1.17 uses java 8, so i can't just delete it i guess?

Reply:
i've got an idea how to "fix it" i'll just list the version names alongside java versions...

like, instead of "Java 8" it'll be "Java 8: 1.14 - 1.17"

do you think this one is better?

Reply:
this sounds great

Reply:
actually it works like this

From [Java Edition 1.12](https://minecraft.fandom.com/wiki/Java_Edition_1.12)([17w13a](https://minecraft.fandom.com/wiki/Java_Edition_17w13a)) to [Java Edition 1.16.5](https://minecraft.fandom.com/wiki/Java_Edition_1.16.5)(1.17: [21w18a](https://minecraft.fandom.com/wiki/Java_Edition_21w18a)), Minecraft requires Java 8 (1.8.0) or newer.[[3]](https://minecraft.fandom.com/wiki/Tutorials/Update_Java#cite_note-3)
From [Java Edition 1.17](https://minecraft.fandom.com/wiki/Java_Edition_1.17)([21w19a](https://minecraft.fandom.com/wiki/Java_Edition_21w19a)) to [Java Edition 1.17.1](https://minecraft.fandom.com/wiki/Java_Edition_1.17.1)(1.18: [1.18 Pre-release 1](https://minecraft.fandom.com/wiki/Java_Edition_1.18_Pre-release_1)), Minecraft requires Java 16 or newer.[[4]](https://minecraft.fandom.com/wiki/Tutorials/Update_Java#cite_note-4)
Since [Java Edition 1.18](https://minecraft.fandom.com/wiki/Java_Edition_1.18)([1.18 Pre-release 2](https://minecraft.fandom.com/wiki/Java_Edition_1.18_Pre-release_2)), Minecraft requires Java 17 or newer.[[5]](https://minecraft.fandom.com/wiki/Tutorials/Update_Java#cite_note-5)

but sometimes newer java versions can't really run old versions

Reply:
In my eyes, that's great. So the, sry for this, noob user will not be confused :D

Reply:
we should make it a new standard 😎